storage/remote/queue_manager_test.go

```diff
@@ -475,7 +475,7 @@ func TestReleaseNoninternedString(t *testing.T) {
 	}
 
 	metric := client_testutil.ToFloat64(noReferenceReleases)
-	require.Equal(t, 0.0, metric, "expected there to be no calls to release for strings that were not already interned: %d", int(metric))
+	require.InDeltaf(t, 0.0, metric, 0.01, "expected there to be no calls to release for strings that were not already interned: %d", int(metric))
```

Comment:
The same way your other changes are using `1.`, you could use `0.` here, it creates a! float64(0)